iacr/2021-08-16-jonas-nick-musig2.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Jonas Nick - MuSig2 (2021-08-16)
+transcript_by: Michael Folkson
+categories: ['conference']
+tags: ['schnorr', 'taproot', 'multisig']
+---
```

Comment:
With the recent merge of https://github.com/bitcointranscripts/bitcointranscripts.github.io/pull/42, `speakers` and `date` are now supported as metadata. I'll open a PR soon to update the README and add the metadata to the rest of the transcripts, but you could also update this PR with
```
speakers: ['Jonas Nick']
date: 2021-08-16
```

Also, with the recent merge of https://github.com/bitcointranscripts/bitcointranscripts/pull/174, When a transcript is added, [@bitcoinwritings](https://twitter.com/bitcoinwritings) will tweet for it and tag speakers & contributors as long as they have been associated with a twitter username at [twitter_handles.json](https://github.com/bitcointranscripts/bitcointranscripts/blob/master/twitter_handles.json).
If you want to tag Jonas Nick you can add an association with
```
#twitter_handles.json
...
"Jonas Nick": "n1ckler"
...
```

Reply:
I think I'll probably wait until you've documented it in the README before I start doing it. It would look like this? Or does the title change to just "MuSig2" now the speaker and the date have their own fields?

---
title: Jonas Nick - MuSig2 (2021-08-16)
transcript_by: Michael Folkson
categories: ['conference']
tags: ['schnorr', 'taproot', 'multisig']
speakers: ['Jonas Nick']
date: 2021-08-16
---

Reply:
Someone could argue that having name and date in the title makes sense because it adds context during discovery:
![before](https://user-images.githubusercontent.com/18506343/180816460-33394b50-f34f-45ed-96ee-6bc6bbc5c1aa.png)

but the next step in my plan is to make some ui changes to use the metadata when displaying those lists:
![after](https://user-images.githubusercontent.com/18506343/180817921-6afe88f1-9224-4633-b77c-f9e07b612974.png)

So I believe that the title should change to just "MuSig2".

Reply:
I was initially thinking no, but that makes sense to keep the functionality until that UI change is implemented and merged.


> ```
> title: Jonas Nick - MuSig2 (2021-08-16)
> transcript_by: Michael Folkson
> categories: ['conference']
> tags: ['schnorr', 'taproot', 'multisig']
> speakers: ['Jonas Nick']
> date: 2021-08-16
> ```

So yes for this format.

Reply:
Ok added in a second commit